azurerm_monitor_action_group/variables.tf

```diff
@@ -1,7 +1,18 @@
-variable "azurerm_monitor_action_group_name" {
-  description = "Descriptive monitor group name"
+# Generic variables
+
+variable "environment" {
+  description = "The nick name identifying the type of environment (i.e. test, staging, production)"
 }
 
+variable "resource_name_prefix" {
+  description = "The prefix used to name all resources created."
+}
+
+# Specific variables for monitor action group
+
+variable "azurerm_monitor_action_group_name_suffix" {
+  description = "Suffix for the monitor action group name."
+
 variable "azurerm_monitor_action_group_short_name" {
```

Comment:
Not sure what's this for...

Reply:
This will generate azurerm_monitor_action_group_name